modules/virt-downloading-virtctl-client.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// virt/install/virt-enabling-virtctl.adoc
+
+[id="virt-downloading-virtctl-client_{context}"]
+= Downloading the virtctl client
+
+Download the `virtctl` client with or without an internet connection using the client provided in a pod in the cluster.
```

Comment:
---This comment is not so important. Please ignore if you don't agree with me---

I think "with or without internet connection" is a little confusing.  I would 

- use  only "Download the `virtcl` client from the cluster" here 
- add a statement into Prerequisites section   "An internet connection is not required. To be able to access the cluster is enough. "

Reply:
After reviewing it, I think "with or without an internet connection" is probably redundant at this point. We don't need to call it out if it works in both scenarios, only if there's an exception. I think calling out this change would be good for a RN, which I see is already in my queue.